### ISSUE TITLE
Reset Mix should not reset output names - ETHOS all

### ISSUE THREAD

**Issue by spoke2570** [FrSkyRC/ETHOS-Feedback-Community#3358]
When a model is switched from one type to another (like cruciform tail to vee tail) it forces a reset of all the mixes. This is fine, but the outputs change names. I find that I frequently want them to keep their original names. As they do retain their custom direction, center, end points maybe they can keep the name, too.

**Comment by RealTadango** [comment]
I do not agree. The names are linked to their function and those need to be reset when you change the type. All other settings might still be valid for the model since they are related to the physical model.

**Comment by mawzthefinn** [comment]
I think the mid-point should be true, only Outputs tied to the core mixes should be reset/renamed. 

Honestly, I'd like a second reset, which is 'Reset only Changed Mixes', so you could do the tail reset only and not affect ailerons, flaps, etc, or vice versa (for when adjusting wing config and not touching tail)

**Comment by strgaltdel** [comment]
wouldn't it be the simplest way to offer the vtail mixer even in a original x tail defined template?
so no need for a reset, 
add the v tail mixer,
delete the dedicated rudder/elev. mixers

just an idea

got myself in the "trap" when i bought an additional v tail fuselage for an existing x tail model..

**Comment by spoke2570** [comment]
I run into this as I convert to the newer canned mixes that work better than the old guys. I expect that I will be changing all of my mixes, but all of the outputs are still connected in my physical model. I want their names and settings. As of now they retain their settings (yay) but not their names, so a lot of checking my backup version to put it right again (load backup model, look, take notes, load the model I’m working on…). Maybe an option to reset or retain output settings?

**Comment by spoke2570** [comment]
Sometimes when I move a model from my hardware matching program to its final development it would be nice to reform the mixing without changing my output settings.

There is already a reset all mixers button in the model menu. Maybe let the user reset and not just force it.

**Comment by Jedsters** [comment]
Maybe I'm missing something here?
As best I can see, there is no difference between a V-Tail Ele mixer and an X-Tail Ele mixer other than V-Tail has an extra channel assigned, ditto Rudder (assuming not doubling up on servos).

To convert an X-Tail to V-Tail is just a case of adding an extra output channel to both mixers and V-Tail to X-Tail just delete a channel from each (making sure you are left with one to each channel).

After that the V-Tail / X-Tail designation in Edit model is largely cosmetic. Unless for some reason you want to define another Ele / Rud in which case the X-Tail mixer gives two channels by default, but again that can be reduced to 1 etc..

As such, isn't there an argument that if you change the tail type in Edit model all it needs to do is change the  number of channels the mixer gives by default. At a push, also reset the values for just those mixers. If I was a betting man I'd wager that's all that Ethos is doing under the covers, changing the number of channels defaulted to by the Ele / Rud mixers according to the Tail type. I notice that with a V-Tail it defaults to 2 channels, but lets you change that to 1/3 etc. not even keeping the numbers even (which is fine by me), which you would think it would do if it was a genuinely different mixer. 

I used to have 2 templates, V and X, but found it tiresome to keep in sync, so now just have 1 and tweek the ele/rud output channels for a V Tail - but mostly I just copy existing models and adjust outputs.

**Comment by spoke2570** [comment]
The V-Tail mixes include differential (which is assignable), X-tail do not. But that is off topic now. 

My particular motivation for not changing output settings is really of little concern. It would just be nice to have the option to choose reset of the other category rather than have it forced.

**Comment by Jedsters** [comment]
Differential does appear to be there when you create an X-Tail, somewhat odd given there is only 1 channel.

![image](https://github.com/FrSkyRC/ETHOS-Feedback-Community/assets/96293240/6723f60b-fd18-4c8b-8196-9a8b8ac92f12)

But yes, choice is always good.

**Comment by spoke2570** [comment]
Try a rudders mix, which is where the differential is really needed in V-Tail. 

The  X-Tail only offers differential to the elevator functions

**Comment by bsongis-frsky** [comment]
For 1.5.1, try to do a little bit better than deleting all mixes!

**Comment by Jedsters** [comment]
Just to include the suggestion from #3419 so it doesn't get lost. May I suggest that when the Wizard creates a V-tail that the control surfaces are called V-Tail1/2 (or similar) rather than Elevators1/2.
Thanks

**Comment by spoke2570** [comment]
The V-Tail mixer is a banger to be sure, but I agree with Jedsters that the nomenclature is bad. Tails, tail, v-tail and rudervator all make better sense in US english, anyway.

**Comment by Jedsters** [comment]
No problem with Tails, tail, v-tail but I'd avoid rudervator for those models which only use the tail as an elevator.  ;)

**Comment by spoke2570** [comment]
Bank and yank, baby...

**Comment by spoke2570** [comment]
This is still an issue, but there is a newer issue posted that is more relevant in the 26 landscape, so closing.
